app/views/interactive_pages/_edit_section.html.haml

```diff
@@ -0,0 +1,48 @@
+:ruby
+  if section_name == 'HEADER_BLOCK'
+    section_hidden_field = hidden_field_tag :section, InteractivePage::HEADER_BLOCK
+    selector_menu = embeddable_selector
+    section_embeddables_list = @page.header_block_visible_embeddables
+  elsif section_name == 'INTERACTIVE_BOX'
+    section_hidden_field = hidden_field_tag :section, InteractivePage::INTERACTIVE_BOX
+    selector_menu = embeddable_interactives_selector
+    section_embeddables_list = @page.interactive_box_visible_embeddables
+  else
+    section_hidden_field = nil
+    selector_menu = embeddable_selector
+    section_embeddables_list = @page.section_embeddables(nil)
+  end
```

Comment:
If `section_name` is nil, in this last case then you can simplify this code. You could put:
`section_embeddables_list =   @page.section_embeddables(section_name)`
outside of the `if,elseif,else` block so each case doesn't need to do that itself.